src/components/Header/TopMainNav/SearchBar/SearchBar.tsx

```diff
@@ -1,10 +1,32 @@
-import React, { useRef, useState } from 'react';
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { useLocation } from '@reach/router';
+import { useSearch } from 'src/hooks';
 import { SearchIcon } from '.';
 
 export const SearchBar = () => {
-  const [searchTerm, setSearchTerm] = useState('');
+  const location = useLocation();
+  const initialSearch = new URLSearchParams(location.search).get('q');
+
+  const [searchTerm, setSearchTerm] = useState(initialSearch?.toString());
   const textInput = useRef<null | HTMLInputElement>(null);
   const focusOnSearchInput = () => textInput.current && textInput.current.focus && textInput.current.focus();
+
+  const {
+    actions: { search },
+  } = useSearch({
+    addsearchApiKey: process.env.GATSBY_ADDSEARCH_API_KEY,
```

Comment:
oh, does this now work on the frontend? (process.env is node runtime, not browser runtime so normally would not work?)

Reply:
no, it's fine. We have Gatsby. it does it's own magic of grabbing env variables and converting them through webpack. 👍